apps/web/src/components/NavBar.svelte

```diff
@@ -14,11 +15,14 @@
     _Logout: () => void
   }
   const _Logout = () => (window as unknown as WindowWithLogout)._Logout()
+
+  const { page } = stores();
+  const linkToCoursePage = ($page.params.courseName && `course/${$page.params.courseName}/`) || "/";
```

Comment:
I think it would be better to use ternary 


```suggestion
  const linkToCoursePage = $page?.params?.courseName
    ? `course/${$page.params.courseName}/`)
    : "/";
```

Reply:
also probably I would call it "homepageLink" or something like that, since it's already not always the course page